src/modules/post/post.controller.ts

```diff
@@ -19,12 +19,11 @@ export class PostController {
   constructor(private readonly postService: PostService) {}
 
   @Post()
-  async addPost(
+  addPost(
```

Comment:
All of the methods you have in post controller in async style.
You have to decide, to write addPost also async or all other method without async.

Reply:
Done